src/Glpi/Console/User/CreateCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
```

Comment:
Adding it in new files is not mandatory (when the code is not copied/pasted from an old file).

```suggestion
```